openassessment/workflow/models.py

```diff
@@ -49,12 +50,14 @@
 # We then use that score as the student's overall score.
 # This Django setting is a list of assessment steps (defined in `settings.ORA2_ASSESSMENTS`)
 # in descending priority order.
-DEFAULT_ASSESSMENT_SCORE_PRIORITY = ['peer', 'self', 'ai']
+DEFAULT_ASSESSMENT_SCORE_PRIORITY = ['staff', 'peer', 'self', 'ai']
 ASSESSMENT_SCORE_PRIORITY = getattr(
```

Comment:
What happens for installations that have already set this setting? I'm wondering if the code should always add 'staff' to the front of the list, regardless of what has been configured, because our UI assumes that staff grades will trump all others.